app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -257,6 +264,37 @@ class NavigationOverlayFragment : Fragment() {
         activity!!.moveTaskToBack(true)
     }
 
+    private fun observeAccountState(): Disposable {
+        val fxaRepo = serviceLocator.fxaRepo
+
+        return fxaRepo.accountState.subscribe { accountState ->
+            when (accountState) {
+                is AccountState.AuthenticatedWithProfile -> {
+                    val fxaProfile = accountState.profile
+                    if (fxaProfile.avatar != null) {
+                        PicassoWrapper.client
+                            .load(fxaProfile.avatar.url)
+                            .into(fxaButton, object : Callback {
+                                override fun onError(e: Exception?) {
+                                    Sentry.capture(e)
```

Comment:
I don't think we need to capture these errors: presumably, some amount of these requests will fail due to network or server errors so the number we receive will be kind of arbitrary.

It could give us value if we notice the number goes from a small amount to a large amount – i.e. everyone is starting to fail requests – but we're not currently equipped to do regression analysis like this. Furthermore, I think we'll be able to catch these types of failures through manual testing with TestRail.